art/attacks/evasion/adversarial_patch/adversarial_patch_tensorflow.py

```diff
@@ -310,14 +338,19 @@ def _random_overlay(
             x_origin_delta = x_origin - x_origin_shifted
             y_origin_delta = y_origin - y_origin_shifted
 
-            a_2 = x_origin_delta - (x_shift / (2 * im_scale))
-            b_2 = y_origin_delta - (y_shift / (2 * im_scale))
+            # a_2 = x_origin_delta - (x_shift / im_scale)
+            # b_2 = y_origin_delta - (y_shift / im_scale)
 
-            transform_vectors.append([a_0, a_1, a_2, b_0, b_1, b_2, 0, 0])
+            # transform_vectors.append([a_0, a_1, a_2, b_0, b_1, b_2, 0, 0])
```

Comment:
Do we need to keep this line?

Reply:
I removed it and added a comment instead.